### ISSUE TITLE
Undefined name in f-string following only blank lines is not reported

### ISSUE THREAD

**Issue by kkurian** [dreadatour/Flake8Lint#115]
**Problem**

*When an f-string follows only blank lines*, an undefined name in the f-string is is not reported.

**Input**

```py
... blank line ...
f'{foo}'
```

**Result**

No errors reported.

**Expected Result**

An error marker on Line 2 reporting `F821 undefined name 'foo'`
